dbms/tests/queries/0_stateless/00974_text_log_table_not_empty.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CURDIR/../shell_config.sh
+
+${CLICKHOUSE_CLIENT} --query="SELECT 6103"
+
+for (( i=1; i <= 50; i++ )) 
```

Comment:
What for 50 retries here with 0.1 delay, if there is `system flush logs`? AFAICS `system flush logs` is sync

Reply:
Because there are too many buffers. Do you have any problems with this test?

Reply:
>Because there are too many buffers.

Sorry don't get it, what buffers?

>Do you have any problems with this test?

Not really (in case of failure it can take pretty long, but that was due to my changes so not a problem)